src/components/app.js

```diff
@@ -1,46 +1,24 @@
-import { Component } from 'preact';
-import { Provider } from 'unistore/preact';
-import createStore from '../store';
-import Routes from './routes';
+import { LocationProvider, ErrorBoundary } from 'preact-iso';
+import { LanguageProvider } from '../lib/i18n';
+// TODO: SolutionProvider should really just wrap the tutorial,
+// but that requires a bit of refactoring
+import { SolutionProvider } from './controllers/tutorial/index.js';
```

Comment:
As noted, the issue is that we need the context _around_ the main tutorial page but at the moment kinda need the router passing the props into it. I figured this works for now, we can refactor & give it a shell later.

It might be beneficial to refactor the routing of the tutorial to sort of match the blog/blog posts.